Client/Frontend/Widgets/ActivityStreamHighlightCell.swift

```diff
@@ -162,7 +162,7 @@ class ActivityStreamHighlightCell: UICollectionViewCell {
         }
 
         self.domainLabel.text = site.tileURL.hostSLD
-        self.titleLabel.text = site.title.characters.count <= 1 ? site.url : site.title
+        self.titleLabel.text = site.title.isEmpty ? site.url : site.title
```

Comment:
This looks like a change in logic?

Reply:
Nice catch, it is an accidental logic change.
Hmm, not sure about the '2 chars is the min title length' logic in that code –I doubt we know for sure that 1 character makes for an invalid title in all languages.
Can I keep this change?

Reply:
Yeah. This logic makes no sense to me. Let's just keep what you have.